client/src/elm/Backend/Update.elm

```diff
@@ -1894,17 +1894,26 @@ updateIndexedDb language currentDate currentTime zscores nurseId healthCenterId
 
                 [ PrenatalBloodGpRsTestRevision uid data ] ->
                     let
-                        ( newModel, extraMsgs ) =
+                        -- We do not catch changes done to model, because
+                        -- it's handled by `processRevisionAndAssessPrenatal`
+                        -- activation that comes below.
```

Comment:
```suggestion
                        -- We do not catch changes done to the model, because
                        -- it's handled by `processRevisionAndAssessPrenatal`
                        -- activation that comes below.
```